### PR TITLE
Add Medium in social media section

### LINKED ISSUE #491: Add Social media icon for Medium
### Actual Behaviour
Add Social media icon for Medium
Also , the links present in the footer lack some links ,which are there in social media section .

### Would you like to work on the issue?
Yes

### PR DESCRIPTION
**Pull request by akshit15013**
- [x] Included a Preview link and screenshot showing after and before the changes.
- [x] Included a description of the change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

- [Preview Link](https://akshit15013.github.io/gci18.fossasia8.org/)

- Fixes #491 

## Description / Changes
Create a PR to add a Social medium section Medium

### After:

- - - - - - - - - - - -

### CODE REVIEW

**Comment by akshit15013** [comment]
I HAVE A MODIFIED CODE, COULD YOU PLEASE MERGE IT?

**Comment by abishekvashok** [comment]
We r not active on medium yet.

**Comment by utkarsh2102** [comment]
Above all, you should really focus on indentation.

**Comment by akshit15013** [comment]
@abishekvashok This means that I cannot add medium

**Comment by STEdirisuriya** [comment]
Did you deleted the repo before this get merged, https://github.com/fossasia/gci18.fossasia.org/pull/490/files

**Comment by Ryan10145** [comment]
@akshit15013 Why is the preview link broken like this? Please fix it: 
image
image

**Comment by akshit15013** [comment]
@ Ryan10145 I do not know what is the problem. I think something wrong has happened with the repo

**Comment by akshit15013** [comment]
@ STEdirisuriya I have closed that PR now. Could you see what is the problem with this PR?

File: index.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
----
```
Comment (pythongiant) [inline index.html]: Why?

**Comment by pythongiant** [comment]
Provide preview link

File: index.html (hunk 1 above)
Comment (akshit15013) [reply index.html]: I have not done this

**Comment by akshit15013** [comment]
@ pythongiant I have provided it but the webpage is not working as desired

File: index.html (hunk 1 above)
Comment (pythongiant) [reply index.html]: Well you have. Its showing up

Comment (akshit15013) [reply index.html]: Let me correct that

Comment (akshit15013) [reply index.html]: I have corrected that

Comment (pythongiant) [reply index.html]: why are you removing it?

**Comment by jogendra** [comment]
Do not add just random links just to finish tasks. I think we have enough social media icons on the website.

**Comment by akshit15013** [comment]
@jogendra Why are you saying that it is unnecessary?

**Comment by jogendra** [comment]
@akshit15013 because we are not active there or even use this medium. So it is totally unnecessary!

**Comment by akshit15013** [comment]
@jogendra Does fossasia has a channel on sina.com?

**Comment by jogendra** [comment]
@akshit15013 No

**Comment by jogendra** [comment]
@akshit15013 I think we have enough social media icons on the website so do not add any unnecessary links, your PR will be close if we found them unnecessary.

**Pull request closed without merging**